forem/forem
# Remove references to article.spaminess_rating

## What type of PR is this? (check all applicable)


## Description

## Related Tickets & Documents

- Related Issue #17222 
- Completes step 1/2 for #17222, where step 2 is creating a migration to remove the `article.spaminess_rating` column

## QA Instructions, Screenshots, Recordings
- verify all tests still pass
- verify articles works as expected

## Added/updated tests?

- [x] No, and this is why: this removes code and specs that are no longer needed

## [Forem core team only] How will this change be communicated?

_Will this PR introduce a change that impacts Forem members or creators, the
development process, or any of our internal teams? If so, please note how you
will share this change with the people who need to know about it._

      [Storybook](https://storybook.forem.com/) (for Crayons components)
      updated. I have filled out the
      [Changes Requested](https://github.com/forem/admin-docs/issues/new?assignees=&labels=&template=changes_requested.md)
      issue template so Community Success can help update the Admin Docs
      appropriately.
      [`CHANGELOG.md`](https://github.com/forem/forem/tree/main/CHANGELOG.md)
      or in a [forem.dev](http://forem.dev) post
      replace this line with details on why this change doesn't need to be
      shared_

## [optional] Are there any post deployment tasks we need to perform?
